cmake/DARTFindBullet.cmake

```diff
@@ -12,9 +12,19 @@
 find_package(Bullet COMPONENTS BulletMath BulletCollision MODULE QUIET)
 
 if((BULLET_FOUND OR Bullet_FOUND) AND NOT TARGET Bullet)
+  if(WIN32 AND "optimized" IN_LIST BULLET_LIBRARIES
+           AND     "debug" IN_LIST BULLET_LIBRARIES)
+    cmake_parse_arguments(BULLET_INTERFACE_LIBRARIES "" "" "debug;optimized"
```

Comment:
Nice! I've only used `cmake_parse_arguments` in writing custom CMake functions but didn't know that this can be used in this way.

Out of curiosity, could we use spaces (i.e., `"debug optimized") instead of semicolons?

Reply:
no, it has to be a list so it needs `;`. I tested first with spaces and it didn't work